index.js

```diff
@@ -9,8 +10,20 @@ const keyRegex = (str) => {
   return new RegExp('^' + str.replace('*','.*'))
 }
 
+let observer;
+
+const observable = new Observable((_observer) => {
+  observer = _observer;
+  return () => {}
+});
```

Comment:
Hmm, looking at the Spec this might be a better implementation?

```JS
let handler

new Observable(observer => {
  handler = data => observer(data);

  return () => { handler = null; }
});

// …

handler({ ... })
```